scrapinghub/__init__.py

```diff
@@ -0,0 +1,10 @@
+__all__ = ['__version__', "APIError", "Connection", "HubstorageClient"]
+
+
+import pkgutil
+__version__ = pkgutil.get_data(__package__, 'VERSION').decode('ascii').strip()
```

Comment:
btw, `__version__` was a string and should remain a string in both  py2 and py3. now it's a unicode in py2.